src/fuzzfetch/core.py

```diff
@@ -498,6 +498,9 @@ def extract_build(self, targets: Sequence[str], path: PathArg = ".") -> None:
         self.resolve_targets(targets)
         path = Path(path)
 
+        if not path.exists():
+            os.mkdir(path)
```

Comment:
This won't work recursively. Is that intended?
```suggestion
        path.mkdir(parents=True, exist_ok=True)
```